projectile.el

```diff
@@ -3342,13 +3351,14 @@ test file."
            "No matching source file found for project type `%s'"
            (projectile-project-type))))
     ;; find the matching test file
-    (let ((test-file (projectile-find-matching-test file-name)))
-      (if test-file
+    (if-let ((overriding-test-file (projectile--test-file-override file-name)))
```

Comment:
Doesn't it make more sense to incorporate this in the existing test lookup function instead of creating the potentially confusing test override concept? I definitely don't like how the code reads now.

Reply:
yeah, in hindsight that makes more sense.